GPU/Common/VertexDecoderX86.cpp

```diff
@@ -222,20 +229,14 @@ JittedVertexDecoder VertexDecoderJitCache::Compile(const VertexDecoder &dec, int
 	// Keep the scale/offset in a few fp registers if we need it.
 	if (prescaleStep) {
 		MOV(PTRBITS, R(tempReg1), ImmPtr(&gstate_c.uv));
-		MOVSS(fpScaleOffsetReg, MDisp(tempReg1, 0));
-		MOVSS(fpScratchReg, MDisp(tempReg1, 4));
-		UNPCKLPS(fpScaleOffsetReg, R(fpScratchReg));
+		MOVAPS(fpScaleOffsetReg, MatR(tempReg1));
```

Comment:
Hmm, I think uv may happen to be aligned but not sure it's guaranteed.  Should we just use MOVUPS to be safe?

-[Unknown]

Reply:
Yeah, okay.